src/test/features/response/form/models/responseType.ts

```diff
@@ -0,0 +1,28 @@
+import { expect } from 'chai'
+import { ResponseType } from 'response/form/models/responseType'
+
+describe('ResponseType', () => {
+
+  describe('all', () => {
+
+    it('should return array of 3', () => {
+      const actual: ResponseType[] = ResponseType.all()
+      expect(actual.length).to.be.eq(3)
+    })
+
+  })
+
+  describe('valueOf', () => {
+
+    it('should return ResponseType', () => {
+      expect(ResponseType.valueOf(ResponseType.OWE_ALL_PAID_NONE.value)).to.be.eq(ResponseType.OWE_ALL_PAID_NONE)
```

Comment:
not sure of the value of this test, especially how its checking all options

Reply:
If we wanna have this text we could have a loop over result of `all` and then make sure `ResponseType.valueOf` can return valid value for every possible 'instance' of ResponseType. Current implementation will give us more maintenance effort.